packages/wonder-blocks-modal/components/focus-trap.js

```diff
@@ -52,19 +52,17 @@ export default class FocusTrap extends React.Component<{children: React.Node}> {
 
     /** Try to focus the given node. Return true iff successful. */
     _tryToFocus(node: Node) {
-        if (typeof node.focus !== "function") {
-            return;
-        }
+        if (node instanceof HTMLElement) {
+            this._ignoreFocusChanges = true;
+            try {
+                node.focus();
```

Comment:
@matchu why would this throw?

Reply:
Not sure, copied this from the WAI-ARIA example and sorta took it for granted, sorry! 😅

We don't make any other attempt to check whether this element is focusable, so I imagine this is a safeguard in case the element is non-focusable.

But it seems like today's browsers silently ignore the request instead of throwing, so maybe this was to work around older browsers that aren't in circulation anymore?